tools/distpartitioning/utils.py

```diff
@@ -182,7 +188,7 @@ def write_metadata_json(metadata_list, output_dir, graph_name):
         graph_metadata["part-{}".format(i)] = metadata_list[i]["part-{}".format(i)]
 
     with open('{}/{}.json'.format(output_dir, graph_name), 'w') as outfile: 
-        json.dump(graph_metadata, outfile, sort_keys=True, indent=4)
+        json.dump(graph_metadata, outfile, sort_keys=False, indent=4)
```

Comment:
The output metadata json, because we set the sort_keys to True, all the entries are alphabetically sorted, by following either the string ordering or numerical ordering. By setting the sort to False all the keys are properly put in their places. And it is easier to read off from this output metadata json